source/spirv_target_env.cpp

```diff
@@ -146,3 +146,28 @@ bool spvParseTargetEnv(const char* s, spv_target_env* env) {
     return false;
   }
 }
+
+bool spvIsVulkanEnv(spv_target_env env) {
```

Comment:
`return env == SPV_ENV_VULKAN_1_0`?

Reply:
What if someone adds SPV_ENV_VULKAN_1_1?

Reply:
Then this code will need to be changed anyway, right? :)

Reply:
I've no strong opinion towards this, so +2'd previously. Your choice. :)

Reply:
The evil plan is that the switch won't compile if you add a new env without putting it either into "Vulkan" or "not Vulkan".

Reply:
That's a good point!